packages/backend-core/src/cache/tests/writethrough.spec.ts

```diff
@@ -17,43 +20,80 @@ describe("writethrough", () => {
   const writethrough = new Writethrough(db, DELAY)
   const writethrough2 = new Writethrough(db2, DELAY)
 
+  const docId = structures.uuid()
```

Comment:
Can we generate a new doc id in the `beforeEach`? So we can ensure these tests won't interfere with each other in future. I could see something like this producing flaky test runs in future

Reply:
These tests are written in a sequential way, so they are already dependent on each other.

Reply:
Is that desirable though? Say I want to run / debug a particular test, it's useful to be able to run it in isolation rather than relying on the suite to pre-populate data for me. It can also be hard to read and reason about a test case where data is being setup in other test executions

Reply:
I think in this case - this is the way the test was written (by me admittedly) - to be honest I think this is a perfectly fine test strategy, as I rarely, if ever run a single test case in isolation, as for very small test cases, I want to check that I haven't broken anything else in the process of trying to fix something. If a test file were to be very large, say taking a minute or so to run then it would be beneficial to be able to run parts of it in isolation, but for tests that are sub-second completion times, and will likely always be that way - I see no real harm in test suites like this.

Reply:
It's more around using the debugger, I run individual tests all the time - when you have 5 or 6 tests all hitting your breakpoint and you need to proceed them all to get to your individual test case, it's tedious. For this reason and the reasons above I think it makes a lot of sense to not have tests cases rely on data from other tests. It's not a safe way of writing them given the order may change in future with refactors. 

I'm fine with leaving this as is given it's existing, but I don't think we should consider it an ok approach for future tests